### ISSUE TITLE
Unable to read text in hidden textbox

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#419]
**Reported by aarnott on 2009-09-18 22:35**
A hidden text control is embedded on the Visual Studio (2008 and 2010 beta 1) Project Conversion dialog (opening page).  Windows Eyes can read it, but NVDA cannot.

**Comment by leonardder** [comment]
Since vs2008 and 2010 have long been superseded by newer versions, these older versions are now obsolete.
